actinia-alpine/Dockerfile

```diff
@@ -27,16 +27,25 @@ USER root
 COPY --from=version /actinia-docker-version.txt /
 
 # # ESA SNAP SETUP
-# ENV LD_LIBRARY_PATH ".:$LD_LIBRARY_PATH"
-# COPY --from=snap /root/.snap /root/.snap
-# COPY --from=snap /usr/local/snap /usr/local/snap
+ENV LD_LIBRARY_PATH ".:$LD_LIBRARY_PATH"
+COPY --from=snap /root/.snap /root/.snap
+COPY --from=snap /usr/local/snap /usr/local/snap
+# commenting out snappy configuration:
 # RUN (cd /root/.snap/snap-python/snappy && pip3 install .)
 # RUN /usr/bin/python3 -c 'from snappy import ProductIO'
 # RUN /usr/bin/python3 /root/.snap/about.py
+# using gpt (graph processing tool) instead:
```

Comment:
If we can do everything with gpt instead of snappy, no need to keep these comments here.